### PR TITLE
Some branding integration, add Windows Symlink

### LINKED ISSUE #35: Add SymLink instructions for PCs
I noticed that the instructions for creating a symlink only work for Macs. Would it be possible to add instructions for PCs? Happy to contribute if needed!

### PR DESCRIPTION
**Pull request by steeleb**
Hey KW, AP, and CC -

Quick review here, I hope. 

KW/AP: please review the updates related to the branding stuff. Katie, please check to make sure that sourcing the style_snippets/ROSS_themes.R script works for you - it should give you the warning "You do not have the Roboto font family installed on your computer, currently using ggplot default text family. See ROSS_themes.R for directions to install the font family on your computer."

CC: I added Symlink instructions for PCs, but I'm not entirely sure it's right. Your review would be helfpul! Closes #35 

You can ignore all the .html, they are just re-renders of the bookdown for the best practices site.

### CODE REVIEW

File: 02-Workflow-Overview.Rmd [hunk 1]
```diff
@@ -72,17 +73,26 @@ you!
...
 ```
 
 The `.` here means that you want that `data` folder to be called `data` as a
 subdirectory at the root of your current .Rproj.
 
+For Windows machines, you *do* have to state what you want the name of the new
+junction to be with the symlink. This should be done in a cmd terminal.
+
+```
+mklink /j "C:/path/to/folder/you/want/to/link" "new-symlink"
```
Comment (ccmothes) [inline 02-Workflow-Overview.Rmd]: I think that you have to specify the full file path of both the existing folder you want to link and where you want to put the new folder. So I would change this to something like:
`mklink /j "C:/path/to/put/new/linked/folder/data" "~/path/to/existing/OneDrive/folder/called/data"`

I would also specify that you cannot already have a folder called 'data/' in the directory you want to place the symlink folder (I forget if that is mentioned above or not).

**Review by ccmothes — APPROVED** [review]
Thanks for adding the PC symlink instructions! I added an edit based on what has worked for me on PCs.

File: 02-Workflow-Overview.Rmd (hunk 1 above)
Comment (kathryn-willi) [reply 02-Workflow-Overview.Rmd]: I agree with Caitlin!

File: style_snippets/ROSS_themes.R [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# store this file in your src/ folder or similar, and then you can automatically
+# load these using `source("style_snippets/ROSS_themes.R")` (as one would source
+# using the current folder structure of this repo)
+
+# download roboto and montserrat from Google Fonts: fonts.google.com
+# open your "FontBook" in applications
```
Comment (kathryn-willi) [inline style_snippets/ROSS_themes.R]:
```suggestion
# open your "Font Book" in applications
```

File: style_snippets/ROSS_themes.R [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+# unzip and drag the contents (the .ttf files) to the "FontBook" (you'll have to
```
Comment (kathryn-willi) [inline style_snippets/ROSS_themes.R]:
```suggestion
# unzip and drag the contents (the .ttf files) to the "Font Book" (you'll have to
```

**Review by kathryn-willi — APPROVED** [review]
Worked for me! Thanks for updating, B!

File: 02-Workflow-Overview.Rmd (hunk 1 above)
Comment (steeleb) [reply 02-Workflow-Overview.Rmd]: y'all are amazing. I think this existed in a previous version of the best practices, and so i was kinda making it up. :)

**Comment by steeleb** [comment]
Added that suggestion, @ccmothes! Thank you both for helping with this!

Merging!

**Pull request merged**